src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/clients/FeesRegisterApi.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.iacaseapi.infrastructure.clients;
+
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.fee.FeeResponse;
+
+@FeignClient(name = "fees-register-api", url = "${fees-register.api.url}")
+public interface FeesRegisterApi {

```

Comment:
We need to add contract testing for this

Reply:
added